packages/suite/src/components/suite/modals/TransactionDetail/components/ChangeFee/index.tsx

```diff
@@ -118,18 +118,19 @@ const ChangeFee = (props: Props & { showChained: () => void }) => {
                                 <Rate>{feeRate}</Rate>
                                 <Amount>
                                     <StyledCryptoAmount>
-                                        <FormattedCryptoAmount value={tx.fee} symbol={tx.symbol} />
+                                        <FormattedCryptoAmount
+                                            disableHiddenPlaceholder
+                                            value={tx.fee}
+                                            symbol={tx.symbol}
+                                        />
                                     </StyledCryptoAmount>
-                                    {tx.rates && (
-                                        <StyledFiatValue>
-                                            <FiatValue
-                                                amount={tx.amount}
-                                                symbol={tx.symbol}
-                                                source={tx.rates}
```

Comment:
It's a bit harder to see as there are two changes mixed in one commit but if I understand this right the removal of `source={tx.rates}` is the change that  implements the

> current fee should use current fiat rate instead of historical fiat rate

right?

Reply:
right

Reply:
Cool. Thx.